x-pack/plugins/ingest_manager/common/types/models/epm.ts

```diff
@@ -234,6 +239,9 @@ export interface Installation extends SavedObjectAttributes {
   es_index_patterns: Record<string, string>;
   name: string;
   version: string;
+  install_status: EpmPackageInstallStatus;
+  install_version: string;
```

Comment:
What's the difference between this and the existing `version`?

Reply:
Perhaps I need a better name, but `version` is the version currently installed whereas `install_version` is version being installed at the moment.  If this is the first time installing a package, they would be the same, but if you are up/down grading to another package it would be different.  it's used to continue the up/down grade when the installation tries again.